django/template/response.py

```diff
@@ -132,17 +145,44 @@ def content(self):
 
     @content.setter
     def content(self, value):
-        """Sets the content for the response
+        """Sets the content for the response.
         """
         HttpResponse.content.fset(self, value)
         self._is_rendered = True
 
 
-class TemplateResponse(SimpleTemplateResponse):
+class RequestContextMixin(object):
     rendering_attrs = SimpleTemplateResponse.rendering_attrs + ['_request']
 
     def __init__(self, request, template, context=None, content_type=None,
             status=None, charset=None, using=None):
-        super(TemplateResponse, self).__init__(
-            template, context, content_type, status, charset, using)
+        super(RequestContextMixin, self).__init__(
+            template, context=context, content_type=content_type,
+            status=status, charset=charset, using=using
+        )
         self._request = request
+
+
+class TemplateResponse(RequestContextMixin, SimpleTemplateResponse):
+    pass
+
+
+class SimpleStreamingTemplateResponse(AbstractTemplateResponse, StreamingHttpResponse):
+    @property
+    def streaming_content(self):
+        """
+        Return a generator for the content given the template and context
+        described by the TemplateResponse.
+        """
+        template = self.resolve_template(self.template_name)
+        context = self.resolve_context(self.context_data)
+        streaming_content = template.render(context, self._request, stream=True)
+        return map(self.make_bytes, streaming_content)
```

Comment:
This will consume the `streaming_content` generator on Python 2. Use `django.utils.six.moves import map` instead.